package.json

```diff
@@ -18,6 +18,9 @@
     "url": "https://github.com/foretagsplatsen/eslint-plugin-cypress/issues"
   },
   "homepage": "https://github.com/foretagsplatsen/eslint-plugin-cypress#readme",
+  "engines": {
+    "node": ">=8.10.0"
```

Comment:
how is this related to this PR?

I have the feeling that the first 3 commits aren't really related to this, or?

Reply:
They are all somewhat related:
- "Allow _ as unused variable name": important because I need such a variable in expect-helpers.js when matching a regular expression
- "Enforce node >= 8.10 to benefit from the regexp 's' flag": I need this flag for the regexp
- "rule table in README": I want the README table to be clean before adding new rules to it